src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -17,10 +16,13 @@ public class PackageExtractionContext
         /// </summary>
         public bool UseLegacyPackageInstallPath { get; set; }
 
+        public PackageSaveModes PackageSaveMode { get; set; }
+
         public PackageExtractionContext()
         {
-            CopySatelliteFiles = false;
+            CopySatelliteFiles = true;
```

Comment:
Actually I see below, but this can break existing codepaths, instead you should set it up when the context is null or pass it as constructor parameter.

Also note that you can in C#6 just do `{ get; set; } = value`